components/brave_wallet_ui/components/shared/external_link/external_link.style.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import Button from '@brave/leo/react/button'
+
+export const LinkButton = styled(Button)<{ fullWidth?: boolean }>`
```

Comment:
Can we use a `width` prop instead to make this component more reusable?
```suggestion
export const LinkButton = styled(Button)<{ width?: string }>`
```